lib/idle_session_expiration_store.rb

```diff
@@ -0,0 +1,15 @@
+require 'redis-session-store'
+
+class IdleSessionExpirationStore < RedisSessionStore
+  def get_session(env, sid)
+    super.tap { update_session_ttl(sid) }
+  end
+
+  def update_session_ttl(sid)
+    expiry = default_options[:expire_after]
+    redis.expire(prefixed(sid), expiry)
+  rescue Errno::ECONNREFUSED, Redis::CannotConnectError => error
+    on_redis_down.call(error, env, sid) if on_redis_down
```

Comment:
Given that the `on_redis_down` option is not set in our session store config, this won't do anything. So, we should either get rid of this section, or actually configure `on_redis_down` to do something, and in that case, we can get rid of `if on_redis_down` because it will be present.